lldb/tools/lldb-dap/tool/lldb-dap.cpp

```diff
@@ -509,6 +547,17 @@ int main(int argc, char *argv[]) {
   }
 
   if (!connection.empty()) {
+    int ttl = 0;
```

Comment:
nit: Could this be `ttl_ms` or maybe a `std::chrono::milliseconds` instead of an `int`?